cabinet/vault.py

```diff
@@ -62,7 +68,49 @@ def add(self, item):
         fname = os.path.join(data_path, fname)
         self._file_write(fname, content)
 
-    def get(self, name):
+    def update(self, name, new_item):
```

Comment:
You're right, we may not want to get "the perfect docs" for each function but we definitely should write some on them.
I wrote this code in a hurry and didn't add proper docstrings, I'll do that now.

Regarding the `del new_item['content']`, we need more comments where logic isn't obvious.
On this case, it may be confusing that the `get` seems to be renamed to `update`, that's a side effect of `get`/`update` being similar on the beginning. The update removes the `content` from the item since from that on it will be the `metadata`, this is more or less the same that we do on `add`. I'll add some comment about this on the code.